src/main/java/org/opensearch/knn/index/query/KNNQueryBuilder.java

```diff
@@ -108,33 +110,33 @@ public KNNQueryBuilder k(Integer k) {
     }
 
     /**
-     * Builder method for max_distance
+     * Builder method for maxDistance
      *
-     * @param max_distance the max_distance threshold for the nearest neighbours
+     * @param maxDistance the maxDistance threshold for the nearest neighbours
      */
-    public KNNQueryBuilder maxDistance(Float max_distance) {
-        if (max_distance == null) {
-            throw new IllegalArgumentException("[" + NAME + "] requires max_distance to be set");
+    public KNNQueryBuilder maxDistance(Float maxDistance) {
+        if (maxDistance == null) {
+            throw new IllegalArgumentException("[" + NAME + "] requires maxDistance to be set");
```

Comment:
Why not `String.format`? Same comment for multiple similar lines where you combine exception messages

Reply:
Thanks will update